pkg/apis/core/types_shoot.go

```diff
@@ -308,10 +308,18 @@ type ETCDEncryptionKeyRotation struct {
 	LastInitiationTime *metav1.Time
 	// LastInitiationFinishedTime is the recent time when the ETCD encryption key credential rotation initiation was
 	// completed.
+	// Deprecated: This field will be removed in a future release. The field will be no longer needed with
```

Comment:
Can we add an _empty line_ between the _docstring_ and the _deprecation_ note ?

```suggestion
	// 
	// Deprecated: This field will be removed in a future release. The field will be no longer needed with
```